app/src/main/java/com/framgia/fsalon/screen/report/reportcustomer/ReportCustomerViewModel.java

```diff
@@ -0,0 +1,154 @@
+package com.framgia.fsalon.screen.report.reportcustomer;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
+import com.framgia.fsalon.BR;
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.utils.formatter.IntIAxisValueFormatter;
+import com.framgia.fsalon.utils.formatter.IntValueFormatter;
+import com.github.mikephil.charting.data.BarEntry;
+import com.github.mikephil.charting.formatter.IAxisValueFormatter;
+import com.github.mikephil.charting.formatter.IValueFormatter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Exposes the data to be used in the Reportcustomer screen.
+ */
+public class ReportCustomerViewModel extends BaseObservable
+    implements ReportCustomerContract.ViewModel {
+    private static String BAR_NEW = "New";
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName